test/integration/camera_plugin.cc

```diff
@@ -120,7 +122,6 @@ INSTANTIATE_TEST_CASE_P(CameraSensor, CameraSensorTest,
 //////////////////////////////////////////////////
 int main(int argc, char **argv)
 {
-  ignition::common::Console::SetVerbosity(4);
```

Comment:
why did you move this ?

Reply:
Oops, I was trying something. Reverted ecf87cffa2fbe4eef2744aa1f1a4320219658ba2